docs/release-notes.rst

```diff
@@ -6,6 +6,177 @@
  Release Notes
 ###############
 
+**************
+ Version 0.35
+**************
+
+Version 0.35.0
+==============
+
+**Release Date:** August 02, 2024
+
+**Breaking Changes**
+
+-  Master Configuration: Replace ``resource_manager.name`` with ``resource_manager.cluster_name``
+   for better clarity and to support multiple resource managers.
+
+   -  Resource managers operate relative to the specific cluster providing resources for Determined
+      tasks, so changing the cluster will affect the resource manager's responses.
+   -  The ``cluster_name`` must be unique for all resource managers when deploying multiple resource
+      managers in Determined.
+   -  During upgrade, replace ``name`` with ``cluster_name`` in the ``resource_manager`` section of
+      your master configuration YAML.
+
+-  Tasks: The :ref:`historical usage <historical-cluster-usage-data>` CSV file has been updated. The
+   header row for slot-hours is now named ``slot_hours`` instead of ``gpu_hours`` to accurately
+   reflect the allocation time for resource pools including those without GPUs. In addition, a new
+   column, ``resource_pool``, has been added to provide the resource pool for each allocation.
+
+-  Cluster: The ``kubernetes_namespace`` field in the resource pool configuration is no longer
+   supported. Users can now submit workloads to specific namespaces by binding workspaces to
+   namespaces using the CLI or API.
+
+-  Cluster: The ``resources.agent_label`` task option and the ``label`` option in the agent config
+   have been removed. Beginning with 0.20.0 release, these options have been ignored. Please remove
+   any remaining references from configuration files and use ``resource_pool`` instead.
+
+**New Features**
+
+-  WebUI/CLI/API: Allow admins to manage resource quotas for auto-created namespaces directly.
```

Comment:
Allow admins to bind namespaces to workspaces and manage resource quotas for auto-created namespaces directly.

Reply:
```suggestion
-  WebUI/CLI/API: Allow admins to bind namespaces to workspaces and manage resource quotas for auto-created namespaces directly.
```